lfucg-exactions/client/js/components/LotForm.js

```diff
@@ -230,9 +230,9 @@ class LotForm extends React.Component {
                                                 </div>
                                                 <div className="col-sm-6">
                                                     <FormGroup label="Permit ID" id="permit_id">
-                                                        <input type="text" className="form-control" placeholder="Permit ID" disabled={currentLot.lot_exactions && currentLot.lot_exactions.current_exactions_number > 0} />
+                                                        <input type="text" className="form-control" placeholder="Permit ID" disabled={currentLot && currentLot.lot_exactions && currentLot.lot_exactions.current_exactions_number > 0} />
                                                     </FormGroup>
-                                                    {currentLot.lot_exactions && currentLot.lot_exactions.current_exactions_number > 0 &&
+                                                    {currentLot && currentLot.lot_exactions && currentLot.lot_exactions.current_exactions_number > 0 &&
```

Comment:
This feels like a lot of work to get a value inline with the UI. I would determine this value above so that you can reduces this to `currentExactions > 0 && ...`

Reply:
Ok- as we progress and get some time I'd definitely love to talk about how things are being rendered. Right now, at least, it's a pattern I've seen Kelly and Jason use to avoid React breaking the rendering when that isn't there.